### DISCUSSION TITLE
Zigbee Sensors Auto Pairing

### DISCUSSION THREAD

**Discussion by AVERioUK** [arendst/Tasmota#15558] [Q&A] [answered]
Hi
I have a sonoff zigbee bridge flashed with the zigbee bridge specific binary and firmware as detailed here:
https://zigbee.blakadder.com/Sonoff_ZBBridge.html 

The issue is that my hub is pairing/listing devices in the webUI as soon as data is received from the sensors, before I have enabled zbpermitjoin 1
I then use ZbForget XXX to remove but they just keep coming back every time data is sent. 

I've never experienced this before - normally only after I have paired devices do they send data to the hub and show in the webUI. 
Does anyone know how to fix this? 
Thanks

**Comment by s-hadinger** [comment, marked as answer]
This happens if the device was already paired. It remembers the network encryption key and rejoins.

You can try ZbLeave to ask the device to leave the network, but there is no guarantee. 

Zbforget only removes from the know list of devices in Tasmota but does not unpair the device

**Reply by AVERioUK** [reply]
Just a follow up Q:
If it remains paired, is it possible to pair the device to a 2nd hub?

**Comment by mcboeck** [comment]
Hi,
I have the same issue. Also installed it like described.
But, I have two Sonoff Bridges. Both are showing up all devices although I joined only one device at a time and not in parallel.
Tried ZbLeave already. No success...
Maybe you have another idea.
Thanks

**Reply by s-hadinger** [reply]
What do you mean you have 2 Sonoff Bridges, and both are showing all devices. A zigbee network can have only 1 coordinator.

Sorry but your issue seems completely different.

**Reply by mcboeck** [reply]
I had two separate sonoff zigbee bridges connected to one mqtt broker. Each should have had its separate sensors and actors.
Unfortunately each sensor was showing up on each device (without pairing mode switched on).
Now i changed one sonoff bridge to a router and everything works like a charm.

**Reply by s-hadinger** [reply]
Are you sure you have different Panids on both coordinators? And different network keys?

**Reply by mcboeck** [reply]
Sorry, now I'm off. I don't know any Panid or network key.

**Reply by sfromis** [reply]
In that case, you may very well have used same for both, which is a bad plan.

The command ZbConfig will show PanID and key. Separate coordinators cannot share these.

**Reply by s-hadinger** [reply]
Also read https://tasmota.github.io/docs/Zigbee/#introduction to know the basics of Zigbee
